controllers/managedocs_controller_test.go

```diff
@@ -507,19 +506,18 @@ var _ = Describe("ManagedOCS controller", func() {
 		}
 	}
 
-	// prometheus tls -- will be uncommented
-	// assertPrometheusTLSConfigValues := func(ctx context.Context,
-	// 	client client.Client) {
-	// 	prom := promTemplate.DeepCopy()
-	// 	Expect(client.Get(ctx, utils.GetResourceKey(prom), prom)).Should(Succeed())
-	// 	tlsConfig := prom.Spec.Web.TLSConfig
-	// 	Expect(tlsConfig.KeySecret.LocalObjectReference.Name).Should(Equal(prometheusServiceSecretName))
-	// 	Expect(tlsConfig.Cert.Secret.LocalObjectReference.Name).Should(Equal(prometheusServiceSecretName))
-	// 	Expect(tlsConfig.ClientCA.ConfigMap.LocalObjectReference.Name).Should(Equal(servingCertCABundleConfigMapName))
-	// 	Expect(tlsConfig.ClientCA.ConfigMap.Key).Should(Equal("service-ca.crt"))
-	// 	Expect(tlsConfig.KeySecret.Key).Should(Equal("tls.key"))
-	// 	Expect(tlsConfig.Cert.Secret.Key).Should(Equal("tls.crt"))
-	// }
+	assertPrometheusTLSConfigValues := func(ctx context.Context,
```

Comment:
- i don't think this is the correct unit test for the functionality
- pls have a re-look at rest of the unit test pieces added in this pr

Reply:
Yes, I didn't actually work on implementing tests for this PR. I'll do it after all the changes are done to the controller.